domrl/agents/big_money_agent.py

```diff
@@ -18,10 +18,31 @@ def find_card_in_decision(decision, card_name):
                 return [idx]
     return [0]
 
+# helper to sort a list of tuples by the second element
+def Sort_List_Of_Tuples(tup):
+    # getting length of list of tuples
+    lst = len(tup)
+    for i in range(0, lst):
+        for j in range(0, lst-i-1):
+            if (tup[j][1] < tup[j + 1][1]):
+                temp = tup[j]
+                tup[j]= tup[j + 1]
+                tup[j + 1]= temp
+    return tup
 
 def get_minimum_coin_card(decision):
-    card_coins = [c.coins for c in decision.moves.card]
-    return [np.argmin(card_coins)]
+    priority = []
+    for i in range(len(decision.moves)):
+        priority.append((decision.moves[i].card.cost, i))
```

Comment:
The more pythonic approach is

`priority = [(move.card.cost, i) for i, move in enumerate(decision.moves)]`

You can further sort this in the same line:

`priority = [(move.card.cost, i) for i, move in enumerate(decision.moves)].sort(key=lambda x: x[1])`